sdcm/nemesis.py

```diff
@@ -1078,9 +1078,12 @@ def disrupt_mgmt_backup(self):
         self._set_current_disruption('ManagementBackup')
         if not self.cluster.params.get('use_mgmt', default=None):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+        if not self.cluster.params.get('backup_bucket_location'):
```

Comment:
in the aws_config.yaml file there are defaults for backup_bucket_location, but in case someone is using a region that is not defined, or using either docker, baremetal or GCE, those values are not set, so we will not be able to run the nemesis, hence skipping.